models/event_handlers/review_submitted_handler.rb

```diff
@@ -21,7 +21,7 @@ def is_approved
   end
 
   def review_preview
-    @data['review']['body'].slice(0...255)
+    (@data['review']['body'] || '+1').slice(0...255)
```

Comment:
Not sure of all the types this could be here, but maybe it's worth doing `@data['review']['body'].presence` to handle the empty string case